app/packages/state/src/recoil/pathData/counts.ts

```diff
@@ -160,7 +175,7 @@ export const cumulativeCounts = selectorFamily<
   get:
     ({ extended, path: key, modal, ftype, embeddedDocType }) =>
     ({ get }) => {
-      return [...new Set(gatherPaths(get, ftype, embeddedDocType))].reduce(
```

Comment:
This functions becomes expensive as the dataset schema grows. The new `gatheredPaths` selector provides caching